core/services/blockhashstore/bhs.go

```diff
@@ -70,7 +72,7 @@ func (c *BulletproofBHS) Store(ctx context.Context, blockNum uint64) error {
 
 		// Set a queue size of 256. At most we store the blockhash of every block, and only the
 		// latest 256 can possibly be stored.
-		Strategy: txmgr.NewQueueingTxStrategy(c.jobID, 256),
+		Strategy: txmgr.NewQueueingTxStrategy(c.jobID, 256, c.config.DatabaseDefaultQueryTimeout()),
```

Comment:
Maybe the 256 should be defined as a constant somewhere?

Reply:
:shrug: Perhaps, but this seems out of scope for this PR.